### ISSUE TITLE
Metric 'disk_total' will lost after 300 secs using gmond default configure

### ISSUE THREAD

**Issue by gassyfeve** [openhpc/ohpc#681]
The following text is the default configure for metric 'disk_total'.
collection_group {
  collect_every = 1800
  time_threshold = 3600
  metric {
    name = "disk_total"
    value_threshold = 1.0
    title = "Total Disk Space"
  }
}
But I found metric 'disk_total' from other nodes will be cleared  automatically after a few miniutes with the following configure in /etc/ganglia/gmond.conf
cleanup_threshold = 300 /*secs */

I always found metric 'disk_total' was lost when I try to fetch xml data from gmond xml port.

**Comment by mrmhodak** [comment]
This is a bug, although it most likely comes from ganglia. Default settings in gmond.conf clear data older than 5 minutes: cleanup_threshold = 300 /*secs */

However, disk_metric data is only collected every 30 minutes: 
collection_group {
  collect_every = 1800
  time_threshold = 3600
  metric {
    name = "disk_total"
    value_threshold = 1.0
    title = "Total Disk Space"
  }
}

Clearly collect_every and time_threshold need to be modified, say, to 180 and 360.

Please advice how to proceed about having this fixed.

**Comment by koomie** [comment]
I assume we can tweak the defaults in the underlying ganglia build. Is `collect_every=180` and `time_threshold=360` agreeable?

**Comment by gassyfeve** [comment]
time_threshold should less than 300

发自我的iphone

在 2018年3月9日，09:07，Karl W. Schulz <notifications@github.com<mailto:notifications@github.com>> 写道：


I assume we can tweak the defaults in the underlying ganglia build. Is collect_every=180 and time_threshold=360 agreeable?

**Comment by mrmhodak** [comment]
OK, lets go with `collect_every=180` and `time_threshold=180`

**Comment by koomie** [comment]
I have created a patch to update these two default settings which will be included in the build for the next v1.3.4 release.   Here is the updated stanza:

```
/* Different than 2.5.x default since the old config made no sense */
collection_group {
  collect_every = 180
  time_threshold = 180
  metric {
    name = "disk_total"
    value_threshold = 1.0
    title = "Total Disk Space"
  }
}
```

Note that because this config file is packaged with `%config(noreplace)` with RPM, it will not overwrite your existing file during an upgrade. Instead, you will see a comment as follows indicating the new file is stored as `gmond.conf.new`

```
warning: /etc/ganglia/gmond.conf created as /etc/ganglia/gmond.conf.rpmnew
```

**Comment by koomie** [comment]
Confirmed presence of updated values in CI results.
